src/include/module/audio/audio_stream.h

```diff
@@ -0,0 +1,53 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2023 Intel Corporation. All rights reserved.
+ *
+ * Author: Karol Trzcinski <karolx.trzcinski@linux.intel.com>
+ *	   Adrian Warecki <adrian.warecki@intel.com>
+ */
+
+#ifndef __MODULE_AUDIO_AUDIO_STREAM_H__
+#define __MODULE_AUDIO_AUDIO_STREAM_H__
+
+#include <stdint.h>
+#include <stdbool.h>
+#include "../ipc/stream.h"
+
+/**
+ * set of parameters describing audio stream
+ * this structure is shared between audio_stream.h and sink/source interface
+ * TODO: compressed formats
+ */
+struct sof_audio_stream_params {
+	enum sof_ipc_frame frame_fmt;	/**< Sample data format */
+	enum sof_ipc_frame valid_sample_fmt;
```

Comment:
can this get a comment as well? or is this a copy

Reply:
This structure was moved as is from [src/include/sof/audio/audio_stream.h](https://github.com/thesofproject/sof/pull/8365/files#diff-0dba4adf6342b75cd53dcd502dc280cf1d47d47fc12685180f6e1b873a7d0825L44)